Source/UI/NewUI/PaymentForm/Default Implementation/BillingFormDetails/Summary/DefaultSummaryCellButtonStyle.swift

```diff
@@ -0,0 +1,16 @@
+import UIKit
+
+public struct DefaultSummaryCellButtonStyle: SummaryCellButtonStyle {
```

Comment:
Can we call this without cell. -> DefaultSummaryButtonStyle ? as merchant can add this ButtonStyle on any button just not a cell button.